spvc/test/run_spirv_cross_tests.py

```diff
@@ -145,17 +189,13 @@ def test_glsl(script_args, shader, filename, optimize):
         flags.append('--separate-shader-objects')
 
     output = None
-    tests = 0
-    passes = 0
     if not '.nocompat.' in filename:
-        tests += 1
         output = spvc(script_args, input, input + filename, flags)
         # logged for compatibility with SPIRV-Cross test script
         log_command(script_args, [script_args.glslang, output])
 
     output_vk = None
-    if '.vk.' in filename or '.asm.' in filename:
```

Comment:
The .asm. case disappeared from here.  Are they gone?  I assume the detailed fail lists will check for any actual problem from this omission,.